pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -405,7 +404,6 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 		// It is safe here to directly use the state to run plugins because we have already initialized the cycle state
 		// for each pending pod when open session and will not meet nil state
 		state := ssn.GetCycleState(task.UID)
-		metrics.Register()
```

Comment:
What does the metrics include? Is it necessary for volcano, or volcano already has these metrics, it's exposed by kube-scheduler framework, may not useful for volcano, we should check that.